### PR TITLE
Reduce logging verbosity for the output filter and Iceberg

### PR DESCRIPTION
**Pull request by bernd**
Switch log statements in `PipelineRuleOutputFilterStateUpdater` from "INFO" to "DEBUG.

Adjust `log4j2.xml` config file to switch verbose Iceberg classes to "warn".

Refs https://github.com/Graylog2/graylog-plugin-enterprise/issues/7703

/nocl

### CODE REVIEW

**Comment by kodjo-anipah** [comment]
@bernd  should set the logs in PipelineRuleOutputFilter to debug to?

**Comment by bernd** [comment]
> @bernd should set the logs in PipelineRuleOutputFilter to debug to?

Yes, good catch. Thanks!

File: graylog2-server/src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -38,7 +38,14 @@
...
         <Logger name="org.apache.parquet.hadoop.InternalParquetRecordReader" level="warn"/>
         <Logger name="org.apache.avro.Schema" level="error"/>
+        <!-- Silence Selenium log chatter -->
         <Logger name="org.openqa.selenium.devtools.Connection" level="warn"/>
+        <!-- Silence Apache Iceberg log chatter -->
+        <Logger name="org.apache.iceberg.BaseMetastoreCatalog" level="warn"/>
+        <Logger name="org.apache.iceberg.BaseMetastoreTableOperations" level="warn"/>
+        <Logger name="org.apache.iceberg.SnapshotProducer" level="warn"/>
+        <Logger name="org.apache.iceberg.SnapshotScan" level="warn"/>
+        <Logger name="org.apache.iceberg.metrics.LoggingMetricsReporter" level="warn"/>
```
Comment (kodjo-anipah) [inline graylog2-server/src/main/resources/log4j2.xml:48]: Found one more
<Logger name="org.apache.iceberg.CatalogUtil" level="warn" />
````
2024-10-02 10:09:30,619 INFO : org.apache.iceberg.CatalogUtil - Manifests to delete: GenericManifestFile{content=DATA, path=/Users/maxwell/workspace/iceberg/local/data-warehouse.streams/66c39d3659b50b74f71f3f1b/metadata/9d2601cd-e39a-495b-b2bb-080987c85670-m0.avro, length=7240, partition_spec_id=0, added_snapshot_id=5943189512678759809, added_data_files_count=1, added_rows_count=2832, existing_data_files_count=0, existing_rows_count=0, deleted_data_files_count=0, deleted_rows_count=0, partitions=[GenericPartitionFieldSummary{contains_null=false, contains_nan=false, lower_bound=[-57, 82, 7, 0], upper_bound=[-57, 82, 7, 0]}], key_metadata=null, sequence_number=279, min_sequence_number=279}, GenericManifestFile{content=DATA, path=/Users/maxwell/workspace/iceberg/local/data-warehouse.streams/66c39d3659b50b74f71f3f1b/metadata/f83e72e5-5ad9-4b9f-907b-a350903810dc-m0.avro, length=7240, partition_spec_id=0, 
````

Comment (bernd) [reply graylog2-server/src/main/resources/log4j2.xml:49]: Thanks! Update pushed.

**Review by kodjo-anipah — APPROVED** [review]
LGTM, thanks Bernd!

**Pull request merged**